.github/workflows/build.yml

```diff
@@ -28,23 +29,31 @@ jobs:
       with:
         repository: ruby/ruby
         path: ruby
+        fetch-depth: 0
```

Comment:
This seems quite slow: 1min10s in https://github.com/DataDog/ruby-dev-builder/actions/runs/13371638547/job/37341247448 vs 4s in https://github.com/ruby/ruby-dev-builder/actions/runs/13377072703/job/37358459749

Would it work if we `fetch-tags: true` without `fetch-depth: 0`?

Another thought is we could simply clone twice, once for master, and once for `ruby_3_4`.
Though with tags it's less easy, would need some other way to find out what's the latest 3.4 release.
From https://github.com/postmodern/ruby-versions/blob/master/ruby/versions.txt maybe but that has some delay.

Reply:
Might also be worth to just schedule another run and see if cloning with `fetch-depth: 0` is always so slow or it was just an unlucky run.

One concern is this extra minute is also spent even if `skip_slow: true`

Reply:
Nice!